engine/src/main/resources/org/hibernate/validator/ValidationMessages.properties

```diff
@@ -25,6 +25,7 @@ org.hibernate.validator.constraints.CreditCardNumber.message        = invalid cr
 org.hibernate.validator.constraints.Currency.message                = invalid currency (must be one of {value})
 org.hibernate.validator.constraints.EAN.message                     = invalid {type} barcode
 org.hibernate.validator.constraints.Email.message                   = not a well-formed email address
+org.hibernate.validator.constraints.ISBN.message                    = invalid ISBN number
```

Comment:
As Gunnar already stated it, let's remove the number and make it `invalid ISBN`.